wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -512,6 +512,7 @@
 "content.system.message_delivered_timestamp" = "Delivered";
 "content.system.message_read_timestamp" = "Seen";
 "content.system.failedtosend_message.general_reason" = "Message could not be sent due to connectivity issues.";
+"content.system.failedtosend_message.user_cancelled_upload_reason" = "Message was not send as the upload was cancelled by the user.";
```

Comment:
It seems like there's a typo:

```suggestion
"content.system.failedtosend_message.user_cancelled_upload_reason" = "Message was not sent as the upload was cancelled by the user.";
```

Also, it feels kind of robotic to say "...was cancelled by the user.". Wouldn't it be more succinct to say something like:

```
upload cancelled
```

or

```
Message was not sent as the upload was cancelled.
```

Reply:
Thanks for seeing the typo 🤦!

Regarding the copy I asked Wolfgang to provide it - the current copy is taken from some designs for android. `Upload cancelled` doesn't work because it might be displayed twice in the view.. See screenshots above. However, I agree `Message was not sent as the upload was cancelled.` is better. I will make a suggestion to Wolfgang.

Reply:
Changed the copy to `Message was not sent as the upload was cancelled.`